roles/install-nfs/files/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```

Comment:
convert this file to a template and replace the values like PATH, image etc.. via ansible variable.

Reply:
done